internal/modules/auth/test.py

```diff
@@ -31,7 +31,7 @@ def test_query_account_fail(self):
         The function `test_query_account_fail` will query wrong account
         and assert fail condition
         """
-        status, _ = query_account("cosmos1xpcfd7pxfmv6gd50y9mwjq50kwqpqrh5mmw72h")
+        status, _ = query_account("umee1xpcfd7pxfmv6gd50y9mwjq50kwqpqrh5mmw72h")
```

Comment:
let's move the address to the function argument, this way we will have more flexibility

Reply:
it is to test the failed query '-' 
The function doesn't have args